integrations/influxdb_connector/influxdb-timestream-connector/src/lib.rs

```diff
@@ -17,13 +24,20 @@ pub mod timestream_utils;
 // that can be made per second is 1.
 pub static TIMESTREAM_API_WAIT_SECONDS: u64 = 1;
 
+// The number of batches processed at the same time.
+// For multi-table multi measure schema, batches are a combination of
+// a table name and a Vec of records bound for that table
+pub static NUM_BATCH_THREADS: usize = 16;
+
 async fn handle_body(
-    client: &timestream_write::Client,
+    client: &Arc<timestream_write::Client>,
```

Comment:
`Arc` is used so that the client can be reused among all threads. `Arc` uses shallow cloning.